browser/ui/views/page_action/brave_page_action_icon_container_view.cc

```diff
@@ -58,6 +63,19 @@ PageActionIconParams& ModifyIconParamsForBrave(PageActionIconParams& params) {
   }
 #endif
 
+#if BUILDFLAG(ENABLE_SPEEDREADER)
```

Comment:
we should drop this buildflag since it defaults to true for all platforms

Reply:
Added a follow up issue https://github.com/brave/brave-browser/issues/37364